jira/client.py

```diff
@@ -480,7 +483,6 @@ def __init__(
             self._create_oauth_session(oauth, timeout)
         elif basic_auth:
             self._create_http_basic_session(*basic_auth, timeout=timeout)
-            self._session.headers.update(self._options["headers"])
```

Comment:
i see this is removed but not added somewhere else ....

Reply:
I removed this as it is already called in L502 (current main) regardless of the auth session type:

https://github.com/pycontribs/jira/blob/db56d5c98dd5a792e6a347c3ff87f7f8cc752599/jira/client.py#L499-L502